pkgs/by-name/cr/create-dmg/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "create-dmg";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "create-dmg";
+    repo = "create-dmg";
+    rev = "refs/tags/${finalAttrs.version}";
```

Comment:
Strange. 

https://github.com/create-dmg/create-dmg/refs/tag/v1.2.2 points to nothing.

Reply:
It does not on github website but the tag is there: https://github.com/create-dmg/create-dmg/tags
If you clone the repo and do `git checkout refs/tags/v1.2.2` it works.

Reply:
So it lacks the `v` before the number.

Reply:
Not sure what you mean, the b is there in the checkout.
Were you maybe commenting on my other response?

Reply:
> Not sure what you mean, the b is there in the checkout.

It should be 

`rev = "refs/tags/v${finalAttrs.version}";`

instead of

`rev = "refs/tags/${finalAttrs.version}";`

Reply:
Yes, that is what o said needed to be changed in the first comment. That of the v is removed from the version attribute, it needs to be added to ref

Reply:
Resolved on the most recent commit, please let me know if this is still an issue!